pkg/i2gw/providers/common/converter.go

```diff
@@ -305,7 +305,9 @@ func (rg *ingressRuleGroup) toHTTPRoute() (gatewayv1beta1.HTTPRoute, field.Error
 	for _, paths := range pathsByMatchGroup {
 		path := paths[0]
 		fieldPath := field.NewPath("spec", "rules").Index(path.ruleIdx).Child(path.ruleType).Child("paths").Index(path.pathIdx)
-		match, err := toHTTPRouteMatch(path, fieldPath)
+		var match *gatewayv1beta1.HTTPRouteMatch
+		var err *field.Error
```

Comment:
any particular reason for changing to two vars instead of `:=`?

Reply:
nope, probably just a leftover from a previous change, I'll revert it.